### ISSUE TITLE
Clarify relation to other project(s) ?

### ISSUE THREAD

**Issue by kevinlong206** [Dids/clover-builder#94]
Greetings,

I found this github from a Google search and downloaded the clover installer from here. After some time of reading /r/hackintosh FAQ, I started thinking I may have downloaded the wrong version of clover, and perhaps even something nefarious.

Your README links to : https://clover-wiki.zetam.org/ for the source project, but that link is broken.

Additionally, it's not clear what the relation if any between this repo and: https://github.com/CloverHackyColor/CloverBootloader/releases

Which I think is the "official" one ? Is this repo automated builds of that one?

Thanks -Kevin

**Comment by Dids** [comment]
The official repository has nothing to do with this one. In fact, the official repository only recently switched from SourceForge to GitHub.

The builds provided in this repository (thanks to `clobber`) are quite different from the official builds, and used to be built for every single commit, but now only build for official releases.

The main goal of these "custom" builds is to add more driver support (or update existing EFI drivers), provide more control over what gets installed (themes for instance) and provide much better default driver selections, compared to the official builds.

TL;DR: Clobber, and these builds, were made to make life easier for people using Clover.
